index.ts

```diff
@@ -115,6 +125,12 @@ export default function install(app: App, {
             return Reflect.get(target, prop);
         }
     });
+    if (slotStart) {
```

Comment:
I think we could combine this into a `$i18nextSlotPattern` (or sth. like this) containing the final regular expression produced via `slotNamePattern()`. Producing the pattern only once might help (a bit) with performance.

We could set `$i18nextSlotPattern` regardless of whether there are custom start/end patterns.

Reply:
I decided to move the `i18nextSlotPattern` (now `slotPattern`) to the global scope, so anybody couldn't access it from the application's global props. What do you think about that?

Reply:
Ah, makes sense. 

This way having different start/end patterns for multiple Vue apps on a page is not possible, but that's a super obscure case. So I think the simplicity of your approach and not "polluting" `globalProperties` is more important than covering this edge case.